omnigibson/objects/controllable_object.py

```diff
@@ -396,7 +407,7 @@ def _postprocess_control(self, control, control_type):
         """
         return control, control_type
 
-    def deploy_control(self, control, control_type, indices=None, normalized=False):
+    def deploy_control(self, control, control_type):
```

Comment:
update doc string

Reply:
tho actually is there any way to preserve the normalized / indices flags here?

Reply:
@cgokmen

Reply:
Why? They weren't getting used for anything afaict. Is this the right place for normalized / indices? When can this be called with those arguments?